bitnami/thanos/templates/query/deployment.yaml

```diff
@@ -159,9 +159,7 @@ spec:
             {{- if .Values.query.grpc.client.serverName }}
             - --grpc-client-server-name={{ .Values.query.grpc.client.serverName }}
             {{- end }}
-            {{- if and .Values.queryFrontend.enabled .Values.queryFrontend.ingress.enabled .Values.queryFrontend.ingress.hostname .Values.queryFrontend.ingress.overrideAlertQueryURL }}
-            - --alert.query-url=http://{{ tpl .Values.queryFrontend.ingress.hostname . }}
-            {{- end }}
```

Comment:
I recommend using a template. Just modify this particular line and keep the rest.
```.diff
-            - --alert.query-url=http://{{ tpl .Values.queryFrontend.ingress.hostname . }}
+            - --alert.query-url={{- template "thanos.ruler.queryURL" .}}
```

Reply:
done with [`974ebf6` (#24456)](https://github.com/bitnami/charts/pull/24456/commits/974ebf6fd65d35748e7b4df9ab097d2c383fa2ad)